Rakefile

```diff
@@ -12,8 +12,8 @@ namespace :style do
   desc 'Run Chef style checks'
   FoodCritic::Rake::LintTask.new(:chef) do |t|
     t.options = { search_gems: true,
-                  fail_tags: %w(correctness rackspace),
-                  chef_version: '11.6.0'
+                  tags: %w(~rackspace-support),
+                  fail_tags: %w(correctness,rackspace)
```

Comment:
This creates an array with one element: `['correctness,rackspace']`. I think that's probably not right.

e.g.
```
irb(main):001:0> %w(a,b)
=> ["a,b"]
irb(main):002:0> %w(a b)
=> ["a", "b"]
```